### PR TITLE
updating tests and setup to work with nemo@2

### PR DESCRIPTION
**Pull request by gabrielcsapo**
```
To https://github.com/gabrielcsapo/nemo-locatex.git
 * [new branch]      update-nemo-2.0.0 -> update-nemo-2.0.0
LM-NYC-00875272:nemo-locatex gacsapo$ grunt
Running "simplemocha:all" (simplemocha) task


  nemo-locatex 
    ✓ should setup with nemo (962ms)
    ✓ should get set up 


  2 tests complete (970 ms)


Running "jshint:files" (jshint) task
>> 2 files lint free.

Done, without errors.
```

### CODE REVIEW

**Comment by grawk** [comment]
Recommend updating the _travis.yml to use node v4 or newer only.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nemo-locatex",
-  "version": "0.4.2",
+  "version": "1.0.0",
```
Comment (grawk) [inline package.json]: https://github.com/paypal/nemo-docs/blob/master/contributing.md#version-tagging

Whoever is going to tag/publish/release note the upcoming version can do this via `npm version major` in order to be consistent in process across nemo modules.

Comment (gabrielcsapo) [inline package.json]: I will revert the package version change @grawk

**Comment by gabrielcsapo** [comment]
Will update this to support nemo@2.1.1 and change the travis version to support node v4.

**Comment by grawk** [comment]
:+1:

**Pull request merged**